tests/test_create.py

```diff
@@ -941,3 +942,17 @@ async def test_preserve_unknown_object_keys(
         assert resource["spec"]["nodes"]["master"]["labels"] == {"s.n.m.l": "1"}
         assert resource["spec"]["nodes"]["master"]["settings"] == {"s.n.m.s": "1"}
         assert resource["status"] == {"foo": "bar", "buz": {"lorem": "ipsum"}}
+
+
+def test_sql_exporter_config():
+    result = get_sql_exporter_config(None, "test-name", None)
+    assert result.metadata.name == "crate-sql-exporter-test-name"
+
+    from os import walk
```

Comment:
import on top of the file

Reply:
isort 5 does not complain about this anymore by default!